daemon/proto/daemon.proto

```diff
@@ -29,6 +33,22 @@ message Team {
   string Id = 1;
 }
 
+message DestroyUserRequest {
+  string username = 1;
+}
+
+message DestroyUserResponse {
+  string message = 1;
+}
+
+message UpdatePasswdRequest{
+  string username = 1;
+  string password = 2;
+}
+
+message UpdatePasswdResponse {
+  string message = 1;
```

Comment:
Add error parameter as well here as well. we have to handle the response on the webclient.

Reply:
Message is used for both success or error which ever comes first.